packages/salesforcedx-vscode-core/package.json

```diff
@@ -96,11 +96,11 @@
       "dist"
     ],
     "packageUpdates": {
-      "main": "dist/index.js",
+      "main": "dist/src/index.js",
+      "salesforceApiVersion": "61",
```

Comment:
I added this config in package.json of the core vsix, in order to provide the missing info from bundling template. So with this setting, one should be aware to update the field when the version of templates is bumped here with a bump of salesforceApiVersion.

I would also like to make this dynamic within esbuild.config.js, so one does not need to update the field frequently, but this added config should be always taken care.

The reason why it works is that the relative path of package.json of [baseGenerator.ts](https://github.com/forcedotcom/salesforcedx-templates/blob/main/src/generators/baseGenerator.ts#L69) within templates library is identical to the relative path of package.json within vsix (of dist/src/index.js).

Reply:
Not a big fan of this. Having this here does not guarantee it is correct. This data should come from templates module.